tests/bulk_create/tests.py

```diff
@@ -39,6 +40,13 @@ def test_efficiency(self):
         with self.assertNumQueries(1):
             Country.objects.bulk_create(self.data)
 
+    @skipUnlessDBFeature('has_bulk_insert')
```

Comment:
The feature flag seems unneeded if there's a vendor check. Is the vendor check needed because this isn't support on other databases?